src/Test.Utilities.Sarif/FileDiffingUnitTests.cs

```diff
@@ -188,24 +188,26 @@ protected static string GenerateDiffCommand(string suiteName, string expected, s
 
         public static bool AreEquivalent<T>(string actualSarif, string expectedSarif, IContractResolver contractResolver = null)
         {
-            if (actualSarif.Equals(expectedSarif)) { return true; }
+            expectedSarif = expectedSarif ?? "{}";
```

Comment:
>expectedSarif [](start = 12, length = 13)

The changes in this method are the fix for #1530, "FileDiffingUnitTests don't notice spurious default-valued properties".